packages/components/src/components/Menu/Menu.tsx

```diff
@@ -63,7 +63,7 @@ const Menu = React.forwardRef<HTMLButtonElement, MenuProps>(
             flexDirection="column"
             maxWidth="280px"
             minWidth="160px"
-            outlineColor="colorBorderPrimary"
+            outlineColor="colorBorderweaker"
```

Comment:
shouldn't this be 
```suggestion
            outlineColor="colorBorderWeaker"
```
?

Did you copy it from storybook? I think we have an issue with token names that have more than two words:

<img width="1005" alt="image" src="https://github.com/user-attachments/assets/b909349d-f9b2-45ef-a104-9a6f36e6bea6">

Also, I would have expected typescript to catch this?

Reply:
Yes I copied it from there. :slightly_smiling_face: